pkg/gatewayserver/observability.go

```diff
@@ -209,26 +218,44 @@ var gsMetrics = &messageMetrics{
 		},
 		[]string{protocol},
 	),
+	downlinkTxAckForwarded: metrics.NewContextualCounterVec(
```

Comment:
This is what `downlinkTxSucceeded` is for

Reply:
This metric relates to TxAck messages that are successfully forwarded to the Network Server (not necessarily meaning that the downlink was sent successfully), so it seems to be different than `downlinkTxSucceeded`.

Reply:
I think that this metric (as well as the event) is very confusing. I actually can't think of any reason why users would be interested in knowing whether the TX ACK was successfully forwarded to the Network Server or not. The only thing they're interested in is the fact that there was a TX ACK in the first place.

Reply:
Indeed. Note that `downlinkTxSucceeded` is also not being used, and I was hoping that this PR would start using it.

Reply:
Hmm, alright. Removed the new metrics and sticking to the existing `downlinkTxSucceeded` and `downlinkTxFailed` metrics.